README.md

```diff
@@ -371,18 +378,40 @@ A short [FAQ](https://tasvideos.org/Bizhawk/FAQ) is provided on the TASVideos wi
 	- For the .NET API, [the ApiHawk server](https://discord.gg/UPhN4um3px)
 - The [TASVideos forum for BizHawk](https://tasvideos.org/Forum/Subforum/64)
 - `#bizhawk` on Libera Chat ([via Matrix](https://matrix.to/#/#bizhawk:libera.chat) or [via IRC](https://libera.chat/guides/connect))
-- The [/r/BizHawk](https://reddit.com/r/BizHawk) subreddit
+- The [/r/BizHawk](https://reddit.com/r/BizHawk) and [/r/TAS](https://reddit.com/r/TAS) subreddits
 
 You can [open a new issue](https://github.com/TASEmulators/BizHawk/issues/new) at any time if you're logged in to GitHub. Please **at the very least read the issue templates**, we tend to ask the same questions for every one-line issue that's opened.
 
 ### Related projects
 
-* [Dolphin](https://dolphin-emu.org) for GameCube and Wii — cross-platform
-* [FCEUX](http://www.fceux.com/web/home.html) for NES/Famicom — cross-platform; TASing is Windows-only
+Unless specified, the below emulators are all available for both Windows and Linux PCs. Many are also available on macOS and Android.
+
+* [Dolphin](https://dolphin-emu.org) for GameCube and Wii
+* [FCEUX](https://fceux.com/web/home.html) for NES/Famicom
+* [GBAHawk](https://github.com/alyosha-tas/GBAHawk) (not part of BizHawk) for GBA — requires Windows host
 * [libTAS](https://github.com/clementgallet/libTAS) for ELF (Linux desktop apps) — requires GNU+Linux host; also emulates other emulators
-* [lsnes](https://tasvideos.org/Lsnes) for GB and SNES — cross-platform
-* [melonDS](http://melonds.kuribo64.net) for Nintendo DS — cross-platform
-* [mGBA](https://mgba.io) for GBA and GB/C — cross-platform
+* [lsnes](https://tasvideos.org/Lsnes) for GB and SNES
+* [openMSX](https://openmsx.org) for MSX
+* Not rerecording-capable:
+	* [ares](https://ares-emu.net) for several consoles
+	* [Cemu](https://cemu.info/) for Wii U
+	* [Flycast](https://github.com/flyinghead/flycast) for Dreamcast
+	* [Ruffle](https://ruffle.rs) for Flash
+	* RIP Ryujinx :(
+* Ported to BizHawk:
+	* [BSNES](https://github.com/bsnes-emu/bsnes) for SNES
+	* [Genesis Plus GX](https://github.com/ekeeke/Genesis-Plus-GX) for Sega's 8- and 16-bit consoles
+	* [GSE](https://github.com/CasualPokePlayer/GSE) for GB/C
+	* [MAME](https://www.mamedev.org) for countless arcades, plug-and-plays, etc.
```

Comment:
Our mame only runs arcades.

Reply:
These are links to upstream. In the cores table, MAME is listed beside "Arcade", which is what we use for both the sysID and long name. We don't really need all these though; #3879 would probably be more beneficial overall.